docs/source/markdown/podman-manifest-push.1.md.in

```diff
@@ -14,6 +14,15 @@ The list image's ID and the digest of the image's manifest.
 
 ## OPTIONS
 
+#### **--add-compression**=*compression*
+
+Makes sure that requested compression variant for each platform is added to the manifest list keeping original instance
+intact in the same manifest list. Supported values are (`gzip`, `zstd` and `zstd:chunked`)
+
+Note: This is different than `--compression` which replaces the instance with requested with specified compression
```

Comment:
That option is named `--compression-format` in Podman.

Reply:
Can we just extend --compression to be a comma separated list.

--compression gzip,zstd

I think that makes for a better User UI.

Reply:
Right now the two have quite different semantics, as the text above says.

Reply:
Not sure why you say that.  
If I say --compression gzip, then it is going to push the image if it has gzip compression or change the compression to gzip if it was something else.
If I push --compression gzip,zstd then I would figure it is going to push gzip and zstd, even if the original was something different.

Reply:
@rhatdan We have already used `--add-compression` since it was overlapping with `--compression` on buildah side, it will be good here for consistency.  

Extending `--compression` changes functionality for example `--compression gzip` will replace instances , but `--compression gzip,zstd` will only replace `gzip` and add `zstd` while changing the order of input will change functionality entirely  consider `--compression zstd,gzip` it will replace `zstd` and add `gzip`. 

We had a prior discussion here: https://github.com/containers/common/pull/1585

Reply:
The _existing_ `--compress` prefers to reuse blobs, even if they don’t use the expected algorithm. That’s a long-standing behavior and I wouldn’t be surprised if users by now relied on it.

(It’s not necessarily the option semantics we would design _now_ as the default, if we were building the CLI from scratch with the feature set of the underlying codebase as of today, but I think the years of past practice change the calculation. @flouthoc is already working on making it possible to force using the requested algorithm independently from the cloning functionality, but that’s not _this_ PR.)